src/pages/using-spark/components/button.mdx

```diff
@@ -18,29 +19,42 @@ An interactive element that represents an action or choice.
 
 ### Usage
 
-Use Buttons to represent the main actions on a page.
+Use buttons to represent what type of actions the users can take on a page and to create a visual hierarchy between different calls to action.
+
+> As a general rule, use no more than two primary call-to-action buttons ([Primary Button](#primary-button)) per page.
 
 Examples of Button usage:
-- To trigger a new UI element to appear on the page.
+- To trigger action.
```

Comment:
'To trigger an action'   
?

Reply:
by adding "an" it might only imply that a button can only one action, when in fact, it usually does more than one.